src/app/services/xml-parsers/generic-parser.service.ts

```diff
@@ -70,15 +70,19 @@ export class GenericParserService {
     return genericElement;
   }
 
-  private parseNote(xml: XMLElement): NoteData {
+  private parseNote(xml: XMLElement): NoteData | GenericElementData {
```

Comment:
When does it returns a `GenericElementData`?

Reply:
Footer notes and note nested in named entity are handled as generic elements, because they should not be rendered with the exponent and the popover. 
But, now that you make me think about it, it may be better to put this information inside the NoteComponent and distinguish the layout there. What do you think?

Reply:
I agree